app/controllers/instructions_controller.rb

```diff
@@ -0,0 +1,9 @@
+class InstructionsController < ApplicationController
```

Comment:
hmm yes. Thanks for pointing out. As there are no additional functionality in this, we can remove this.